src/js/renderShabad.js

```diff
@@ -91,6 +91,27 @@ function metaData(data, nav) {
 }
 
 function Baani(gurbani) {
+  const onCopyClick = ({ currentTarget }) => {
+    const textarea = currentTarget.parentNode.parentNode.querySelector('textarea');
+    textarea.style.display = 'block';
+    textarea.focus();
+    textarea.select();
+    document.execCommand('copy');
+    textarea.blur();
+    textarea.style.display = 'none';
+  };
+
+  const onTweetClick = ({ currentTarget}) => {
+    let tweet = currentTarget.parentNode.parentNode.querySelector('textarea').value;
+    const shortURL = `\n${shortenURL()}`;
+    if (tweet.length + shortURL.length > 134) {
+      tweet = `${tweet.substring(0, 132 - shortURL.length)}..`;
```

Comment:
@navdeepsinghkhalsa can we move this to 280 - 8 now, as twitter has increased the limit.

Reply:
Yes, please. Will await that change before merging